<launch>
 <param name = "product" type = "string" value = "LD09" />
 <node name="LD09" pkg="ldlidar" type="ldlidar" output="screen">
 </node>
 
</launch>